src/main/scala/edu/twitter/classification/Classifier.scala

```diff
@@ -27,41 +26,21 @@ class Classifier(ssc: StreamingContext) {
     *
     * @return stream of `ClassifiedTweets`
     */
-  def createClassifiedStream(): DStream[ClassifiedTweet] = {
+  def createClassifiedStream(genericModelBuilder: GenericModelBuilder): DStream[ClassifiedTweet] = {
```

Comment:
Can you add parameters in the documentation?

Reply:
Sure, will do it.